### PR TITLE
Added a new cheatsheet of React Js

### PR DESCRIPTION
**Pull request by Rohitsuper69**
### Checklist:
- [x] Read our [contributing guidelines](../docs/CONTRIBUTING.md).
- [x] Search for duplicates.
- [x] Used an informative name for this pull request.

### Follow-up
- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Rohitsuper69** [comment]
Is this rejected or being reviewed?

**Comment by Rohitsuper69** [comment]
Hey are you even checking?

File: CheatSheets/reactjs-cheatsheet.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+---
+layout: ../../layouts/BlogLayout.astro
+title: "ReactJS Cheatsheet"
+---
```
Comment (crescentpartha) [inline CheatSheets/reactjs-cheatsheet.md:4]:
```suggestion
--- 
title: "ReactJS Cheatsheet"
description: "..."
created: date
---
```

File: CheatSheets/reactjs-cheatsheet.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+## Create React App
```
Comment (crescentpartha) [inline CheatSheets/reactjs-cheatsheet.md:6]: Add `Table of Contents` and link all the headings here.

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Modification Requirements:
1. Rename your file from `reactjs-cheatsheet.md` to `react-cheatsheet.md`
2. Add `Table of Contents`
3. Add `title`, `description`, and `created` in the top of the sheet.
4. Follow the [CheatSheet-Template](https://github.com/crescentpartha/CheatSheets-for-Developers/blob/main/docs/CheatSheet-Template.md)

**Review by Rohitsuper69 — COMMENTED** [review]
Done changes

**Review by crescentpartha — APPROVED** [review]
Approved your changes.

**Pull request merged**